Dockerfile

```diff
@@ -1,23 +1,22 @@
 # syntax=docker/dockerfile:1.4
-FROM buildpack-deps:bookworm AS builder-nsjail
+FROM buildpack-deps:trixie AS builder-nsjail
 
 WORKDIR /nsjail
 
 RUN apt-get -y update \
     && apt-get install -y --no-install-recommends \
-        bison\
+        bison \
         flex \
-        libprotobuf-dev\
+        libprotobuf-dev \
         libnl-route-3-dev \
         protobuf-compiler \
     && rm -rf /var/lib/apt/lists/*
 
-RUN git clone -b master --single-branch https://github.com/google/nsjail.git . \
-    && git checkout dccf911fd2659e7b08ce9507c25b2b38ec2c5800
```

Comment:
I think we used a commit instead of a branch to minimise the amount of data we pull from the remote.

Reply:
Wouldn't doing `-b master--single-branch` clone the entire master branch and then after that we checkout to a older commit.

Whereas the `-b 3.4 --depth 1` only clones the single commit that is tagged with that name?

happy to change it back if I've misunderstood that

Reply:
Sorry, I had it backwards. We originally had it as you wrote it now. It was changed to a commit because we wanted a specific unreleased version. 

See the commit below and its associated PR description. https://github.com/python-discord/snekbox/commit/b1bcd3b04fb2a83fd7bd4fe930582ccf0b6bf627

Reply:
Ah makes sense!